app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  def update_comment_counter
```

Comment:
[OPTIONAL]
- You can use **rails active callbacks** like I mentioned in a comment earlier.